src/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2023 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { columns, data, rowKey } from '../Table/Table.mocks'
+import { Button } from '../Button'
+import { Card } from '.'
+import { Table } from '../Table'
+import { actionButton } from './Card.mocks'
+
+const meta = {
+  component: Card,
```

Comment:
we should disable the extra control to avoid this

Reply:
![image](https://github.com/mia-platform/design-system/assets/9254840/2c87fc7b-8eef-44eb-aa47-9af645ce2602)

Reply:
Should we do the same for `children` as well? Otherwise, the same problem occurs in content-related stories

Reply:
Disabling them, however, makes them disappear from the Storybook documentation, so I don't know if that is exactly what we want 😬

Reply:
I found out the following:
- `argTypes: { foo: { table: { disable: true } } }` hides the prop `foo` from the whole UI ❌
- `argTypes: { foo: { control: false } }` hides the controls but does not remove the prop `foo` from the documentation ✅